src/components/Carousel.js

```diff
@@ -90,11 +90,11 @@ export default class Carousel extends Component {
 
     // adding event listeners for swipe
     if (this.node) {
-      this.node.ownerDocument.addEventListener('mousemove', this.onMouseMove, true);
-      this.node.ownerDocument.addEventListener('mouseup', this.onMouseUpTouchEnd, true);
-      this.node.ownerDocument.addEventListener('touchstart', this.simulateEvent, true);
-      this.node.ownerDocument.addEventListener('touchmove', this.simulateEvent, { passive: false });
-      this.node.ownerDocument.addEventListener('touchend', this.simulateEvent, true);
+      this.node.parentElement.addEventListener('mousemove', this.onMouseMove, true);
```

Comment:
thanks for this PR;
could you explain how this fixes the problem?
are you sure it's enough to add an event listener to only the parent element but not both to the parent element and the owner document?

Reply:
The owner document references window.document which causes touch behavior issues. In my case, it was when I was using google maps. Why would I simulate touch events in a global context which can overlap with other libs? I want to keep it as local as possible. 
Reference: https://stackoverflow.com/questions/9845043/when-node-ownerdocument-is-not-window-document